components/infobox/wikis/pubg/infobox_league_custom.lua

```diff
@@ -0,0 +1,224 @@
+	['4v4'] = '4v4 TDM[[Category:4v4 TDM Tournaments]]',
+	['war mode'] = 'War Mode',
+	default = '[[Category:Unknown Mode Tournaments]]',
+}
+_MODES.solos = _MODES.solo
+_MODES.duos = _MODES.duo
+_MODES.squads = _MODES.squad
+_MODES.tdm = _MODES['2v2']
+
+local _PERSPECTIVES = {
+	fpp = {'FPP'},
+	tpp = {'TPP'},
+	mixed = {'FPP', 'TPP'},
+}
+_PERSPECTIVES.first = _PERSPECTIVES.fpp
+_PERSPECTIVES.third = _PERSPECTIVES.tpp
+
+local _PLATFORMS = {
+	pc = '[[PC]][[Category:PC Competitions]]',
+	mobile = '[[Mobile]][[Category:Mobile Competitions]]',
+	newstate = '[[New State]][[Category:Mobile Competitions]]',
+	pclite = '[[PC LITE]][[Category:PC Competitions]][[Category:PC LITE Competitions]]',
+	peace = '[[Peacekeeper Elite|Peace Elite]][[Category:Peacekeeper Elite Competitions]][[Category:Mobile Competitions]]',
+	bgmi = '[[Battlegrounds Mobile India|BGMI]]' ..
+		'[[Category:Battlegrounds Mobile India Competitions]][[Category:Mobile Competitions]]',
+	console = '[[Console]][[Category:Console Competitions]]',
+	default = '[[Category:Unknown Platform Competitions]]',
+}
+_PLATFORMS.lite = _PLATFORMS.pclite
+
+function CustomLeague.run(frame)
+	local league = League(frame)
+	_args = league.args
+
+	league.addToLpdb = CustomLeague.addToLpdb
+	league.createWidgetInjector = CustomLeague.createWidgetInjector
+	league.defineCustomPageVariables = CustomLeague.defineCustomPageVariables
+
+	return league:createInfobox(frame)
+end
+
+function CustomLeague:createWidgetInjector()
+	return CustomInjector()
+end
+
+function CustomInjector:parse(id, widgets)
+	if id == 'gamesettings' then
+		return {
+			Cell{name = 'Game version', content = {
+					CustomLeague._getGameVersion()
+				}
+			},
+			Cell{name = 'Game mode', content = {
+					CustomLeague:_getGameMode()
+				}
+			},
+			Cell{name = 'Platform', content = {
+					CustomLeague:_getPlatform()
+				}
+			},
+		}
+	elseif id == 'prizepool' then
+		return {
+			Cell{
+				name = 'Prize pool',
+				content = {CustomLeague:_createPrizepool()}
+			},
+		}
+	elseif id == 'liquipediatier' then
+		return {
+			Cell{
+				name = 'Liquipedia tier',
+				content = {CustomLeague:_createTierDisplay()},
+				classes = {_args['pubgpremier'] == 'true' and 'valvepremier-highlighted' or ''},
+			},
+		}
+	elseif id == 'customcontent' then
+		if _args.player_number then
+			table.insert(widgets, Title{name = 'Players'})
+			table.insert(widgets, Cell{name = 'Number of players', content = {_args.player_number}})
+		end
+
+		--teams section
+		if _args.team_number then
+			table.insert(widgets, Title{name = 'Teams'})
+			table.insert(widgets, Cell{name = 'Number of teams', content = {_args.team_number}})
+		end
+	end
+	return widgets
+end
+
+function CustomLeague:addToLpdb(lpdbData, args)
+	lpdbData.game = _game or args.game
+	lpdbData.participantsnumber = args.player_number or args.team_number
+	lpdbData.extradata = {
+		individual = String.isNotEmpty(args.player_number) and 'true' or '',
+	}
```

Comment:
Should publishertier be added to lpdb also?

Reply:
Is it only for where there is multiple tier of events labelled by the publisher themself?

As of now PUBG there isnt any existence of Publisher Tier (like Ubisoft Tier or Valve Tier) just only the yellow-highlighted premier event are the furthest of it

Reply:
It's being set as a wiki-variable here however https://github.com/Liquipedia/Lua-Modules/pull/1041/files#diff-f73a7d8b232fba73d5591ad133846d1a1cba82be1ffee5d226461b2b26b1cd42R137

Reply:
ah, ok. Yeah then would be okay to have it, how should I code it in

is it similar to these 4 like ( e.g. **lpdbData.publishertier = tournament_publishertier** ? )

Reply:
I have try add it, you can check if thats the right way for me to wrote it

(as of now, the **Publisher Tier** section when Browse LiquipediaDB data on a tournament is still empty if thats anything useful)

Reply:
Looks good